modules/ROOT/pages/mule-message-encryption-processor.adoc

```diff
@@ -39,13 +39,14 @@ image::encrypt.png[]
 +
 [%header%autowidth.spread]
 |===
-|Field |Req'd |Value
+|Field |Required |Value
 |*Display Name* |  |A unique name for your message processor.
-|*Config Reference* |x |Use the drop down to select the global Encryption element you created.
+|*Connector Configuration* |x |Use the drop down to select the global Encryption element you created.
 |*Operation* |x |Encrypt
 |*Input reference* |  |Enter a Mule expression to define the part(s) of the message Mule should encrypt. If no value is entered, Mule encrypts the entire message payload.
+|*Stream Name* | |
```

Comment:
```suggestion
|*Stream Name* | name of the decrypted payload in a PGP encrypted file. In case a file is encrypted using Mule, this shall help the user keep the original filename so that it is recovered on decryption of the resulting PGP file |
```